packages/vite/src/executors/test/lib/utils.ts

```diff
@@ -59,7 +61,9 @@ export async function getOptions(
       ? process.cwd()
       : relative(context.cwd, joinPathFragments(context.root, projectRoot));
 
-  const normalizedExtraArgs = normalizeArgs(extraArgs);
+  const { parseCLI } = await loadVitestDynamicImport();
+
+  const normalizedExtraArgs = parseCLI(getOptionsAsArgv(options));
```

Comment:
CI is failing because the `parseCLI` helper expects the `vitest` as the first argument.